hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/RowCounter.java

```diff
@@ -137,6 +137,103 @@ public Job createSubmittableJob(Configuration conf) throws IOException {
     return job;
   }
 
+  /**
+   * Sets up the actual job.
+   *
+   * @param conf  The current configuration.
+   * @param args  The command line parameters.
+   * @return The newly created job.
+   * @throws IOException When setting up the job fails.
+   * @deprecated please use main method instead.
```

Comment:
How about making the comment include some more details about the deprecation? Something like `As of release 2.3.0, this will be removed in HBase 4.0.0.`

Reply:
So @ndimiduk  we cannot remove it in release `3.0.0`, only on `4.0.0`?

Reply:
If it is deprecated in 2.3.0 we need a whole major release cycle to remove it. The next one then would be 4.0.0.

Reply:
> So @ndimiduk we cannot remove it in release `3.0.0`, only on `4.0.0`?

Correct, that's my read of our community assurance.